pkg/config.go

```diff
@@ -0,0 +1,83 @@
+package pkg
```

Comment:
if you are going to use a `pkg` dir. this file should really be at `pkg/config/config.go` as it's own config package rather than trying to maintain a monolithic package package.
```suggestion
package config
```

Reply:
Since this is a standalone application and not a library I don't know if I see much point in having both `/internal` and `/pkg` For this project I would say just pick one or the other (or neither). But imo not both